### PR TITLE
feat: expose more details in trivy_vulnerability_id metric

### LINKED ISSUE #152: Extension vulnerabilityreport for prometheus metrics
For request „[expose findings as prometheus metrics #78](https://github.com/aquasecurity/trivy-operator/issues/78)“ was create a metric for vulnerabilityreport.

This metric is suitable for a Dashborad overview is count and risk of a Deployment. 

But there are no Information which kind of vulnerability is relatet to this deployment.

For a more descriptive Dashbord it would be nice to have more Information like

•	Score
•	Title
•	Vulnerability ID

### PR DESCRIPTION
**Pull request by MiggL**
## Description
Extends the `trivy_vulnerability_id` metric to include `fixed_version`, `vuln_title`, and `vuln_score`

## Related issues
- Close #152 

## Checklist
- [x] I've read the [guidelines for contributing](https://github.com/aquasecurity/trivy-operator/blob/main/CONTRIBUTING.md) to this repository.
- [x] I've added tests that prove my fix is effective or that my feature works.
- [x] I've not updated the [documentation](https://github.com/aquasecurity/trivy-operator/tree/main/docs).

### CODE REVIEW

**Comment by dkulchinsky** [comment]
@MiggL this is cool, I was going to suggest a similar one (was also thinking that `trivy_image_exposedsecrets` could be useful with a few more labels)

you'll probably want to adjust the tests in https://github.com/aquasecurity/trivy-operator/blob/main/pkg/metrics/collector_test.go accordingly, with the new labels.

**Comment by chen-keinan** [comment]
@MiggL thank you for the contribution I have added two comments

**Comment by chen-keinan** [comment]
@MiggL also need to update [metric collector test](https://github.com/aquasecurity/trivy-operator/blob/main/pkg/metrics/collector_test.go#L157) with new fields and [docs](https://github.com/aquasecurity/trivy-operator/blob/main/docs/tutorials/integrations/metrics.md)

**Comment by MiggL** [comment]
@chen-keinan I've updated the metric collector test and docs now.

> @MiggL thank you for the contribution I have added two comments

What comments are you referring to?

**Comment by chen-keinan** [comment]
@MiggL that's it nothing else, just please rebase your branch with upstream.

File: pkg/metrics/collector.go [hunk 1]
```diff
@@ -430,11 +437,18 @@ func (c ResourcesMetricsCollector) collectVulnerabilityIdReports(ctx context.Con
...
+					vulnLabelValues[12] = NewSeverityLabel(vuln.Severity).Label
+					vulnLabelValues[13] = vuln.PackageType
+					vulnLabelValues[14] = vuln.Class
+					vulnLabelValues[15] = vuln.VulnerabilityID
+					vulnLabelValues[16] = vuln.Title
+					if vuln.Score != nil {
+						vulnLabelValues[17] = strconv.FormatFloat(*vuln.Score, 'f', 1, 64)
+					} else {
+						vulnLabelValues[17] = ""
+					}
```
[the remark this answers is not in the document]
Comment (chen-keinan) [reply pkg/metrics/collector.go]: @MiggL just this nit: comment

**Comment by MiggL** [comment]
All right. Fyi I also changed the third strconv.FormatFloat argument to -1 so that the string contains the least amount of decimals i.e. "6" instead of "6.0" but still "6.1"

**Comment by MiggL** [comment]
@chen-keinan fixed

**Comment by chen-keinan** [comment]
@MiggL LGTM 🚀

**Review by chen-keinan — APPROVED** [review]

**Pull request merged**